packages/x-charts/src/context/DrawingProvider.tsx

```diff
@@ -30,6 +30,11 @@ export const DrawingContext = React.createContext<DrawingArea>({
 });
 export const SVGContext = React.createContext<React.RefObject<SVGSVGElement>>({ current: null });
 
+/**
+ * API:
+ *
+ * - [DrawingProvider API](https://mui.com/x/api/charts/drawing-provider/)
```

Comment:
Why not, but there is a lot to do for those pages to be readable 🫣

Reply:
I know ... but we have it in place once its ready, aye? 🏴‍☠️

Reply:
Ho yes, I agree with putting them. it was more to be sure you're also aware about it :)